apps/geo/views.py

```diff
@@ -241,5 +242,43 @@ def get(self, request, version=None):
             if not region.geo_options:
                 region.calc_cache()
             result[str(region.id)] = region.geo_options
-
         return response.Response(result)
+
+
+class GeoAreaView(viewsets.ReadOnlyModelViewSet):
+    permissions_classes = [permissions.IsAuthenticated]
+    serializer_class = GeoAreaSerializer
+
+    def get_queryset(self):
+        queryset = GeoArea.objects.prefetch_related(
+            'admin_level',
+            'admin_level__region'
```

Comment:
We don't need prefetch_related if we are getting the values from annotation.

Also, Note:
- Prefetch related is used if there are multiple objects for that field (this will separately prefetch in bulk)
- We use select related for single objects. (this will do join)